sql/parser/Makefile

```diff
@@ -1,5 +1,6 @@
-.PHONY: all
-all: sql.go keywords.go reserved_keywords.go
+TARGETS = sql.go keywords.go reserved_keywords.go
+.PHONY: all $(TARGETS)
```

Comment:
The targets (`{sql,keywords,reserved_keywords}.go`) are not phony. They are real files for which we want make to compare the timestamps. I missed the discussion of why this is necessary. Care to elaborate? If I stumbled across a Makefile containing the above I would fix it, so a comment is definitely necessary to explain this peculiar usage of `.PHONY`.